### PR TITLE
Subtitle for step two

### PR DESCRIPTION
**Pull request by LomyW**
for now: can't query match appointments, thats why we skip step two at the moment.

### CODE REVIEW

File: src/lang/appointment/de.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ const appointment = {
     },
     create: {
         assignmentHeader: 'Für welches Lernangebot soll dieser Termin erstellt werden?',
+        insightMatchHeader: 'Deine bestehenden Termine mit ',
+        insightCourseHeader: 'Deine bestehenden Termine für ',
```
Comment (timidak) [inline src/lang/appointment/de.ts:14]: Use a variable in i18n for the name of the course or learning buddy. This approach might work for german and english but probably won't for other languages.

File: src/pages/create-appointment/AppointmentsInsight.tsx [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+import { Box, Button, Stack, useBreakpointValue, Text } from 'native-base';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { useQuery } from '@apollo/client';
+import { gql, useQuery } from '@apollo/client';
 import { useLayoutHelper } from '../../hooks/useLayoutHelper';
 import AppointmentList from '../../widgets/appointment/AppointmentList';
-import { gql } from '../../gql';
 import CenterLoadingSpinner from '../../components/CenterLoadingSpinner';
+// import CenterLoadingSpinner from '../../components/CenterLoadingSpinner';
```
Comment (timidak) [inline src/pages/create-appointment/AppointmentsInsight.tsx:8]: remove this comment

File: src/pages/create-appointment/AppointmentsInsight.tsx [hunk 3]
```diff
@@ -49,7 +49,22 @@ const AppointmentsInsight: React.FC<Props> = ({ next, back }) => {
...
+                    <Text>
+                        {t('appointment.create.insightCourseHeader')}
+                        <Text fontWeight="bold">{data?.subcourse?.course?.name ? data?.subcourse?.course?.name : 'Kursname'}.</Text>
+                    </Text>
+                </Box>
+            ) : (
+                <Stack direction="row" py={6}>
+                    <Text>
+                        {/* // TODO add match partner name */}
+                        {t('appointment.create.insightMatchHeader')} <Text fontWeight="bold">Leon Jackson.</Text>
```
Comment (timidak) [inline src/pages/create-appointment/AppointmentsInsight.tsx:64]: Use the variable here

File: src/pages/create-appointment/AppointmentsInsight.tsx [hunk 4]
```diff
@@ -49,7 +49,22 @@ const AppointmentsInsight: React.FC<Props> = ({ next, back }) => {
     // TODO add empty state from upcoming story
     return (
         <Box>
-            {/* {loading && <CenterLoadingSpinner />} */}
+            {loading && <CenterLoadingSpinner />}
+            {isCourse ? (
+                <Box py={6}>
+                    <Text>
+                        {t('appointment.create.insightCourseHeader')}
+                        <Text fontWeight="bold">{data?.subcourse?.course?.name ? data?.subcourse?.course?.name : 'Kursname'}.</Text>
```
Comment (timidak) [inline src/pages/create-appointment/AppointmentsInsight.tsx:57]: use a variable here, too.

**Review by timidak — CHANGES_REQUESTED** [review]
Some minor changes, rest looks fine to me 👍🏻

File: src/lang/appointment/de.ts (hunk 1 above)
Comment (LomyW) [reply src/lang/appointment/de.ts:14]: is there a way to make only one part of the translation bold?

**Review by timidak — APPROVED** [review]
Nice work 🎉

**Pull request merged**